src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -575,6 +586,15 @@ void FullyConnected::createDescriptor(const std::vector<MemoryDescPtr> &inputDes
                              MemoryDescUtils::convertToDnnlMemoryDesc(outDesc)->getDnnlDesc());
 }
 
+void FullyConnected::createPrimitive() {
+    Node::createPrimitive();
+    // WA: We update implType to know whether weights decompression was used inside the kernel
+    NodeDesc *selected_pd = getSelectedPrimitiveDescriptor();
```

Comment:
Impl type should be update inside prepareParams() function to satisfy dynamism flow.